README.rst

```diff
@@ -803,7 +803,7 @@ keys.
 You can host your own copy of Sydent, but this will prevent you reaching other
 users in the Matrix ecosystem via their email address, and prevent them finding
 you. We therefore recommend that you use one of the centralised identity servers
-at ``https://matrix.org`` or ``https://vector.im`` for now.
+at ``https://matrix.org`` or ``https://riot.im`` for now.
```

Comment:
I'm not sure if there's actually an identity server running on riot.im

Reply:
vector.im is correct and this seems to be unrelated to the subject of the pr.